src/arch/Cargo.toml

```diff
@@ -8,8 +8,7 @@ edition = "2018"
 kvm-bindings = { git = "https://github.com/firecracker-microvm/kvm-bindings", tag = "v0.2.0-2", features = ["fam-wrappers"] }
 kvm-ioctls = { git = "https://github.com/firecracker-microvm/kvm-ioctls", tag = "v0.5.0-2" }
 libc = ">=0.2.39"
-vm-memory = { version = ">=0.2.2", features = ["backend-mmap"] }
-
+vm-memory-proxy = { path = "../vm-memory-proxy" }
```

Comment:
Can we keep the old name (`vm-memory`) ? Chaning it has 2 disadvantages:
1. It's very verbose
2. We'll have to undo this change when we switch back to the `vm-memory` crate.

Reply:
I agree on both points. My idea here was to make the switch be explicit and much clear.

Reply:
I've updated the PR and now the proxy crate has the same name: `vm-memory`

Reply:
Only 11 files changed now as opposed to the previous version with 50 files changed 👍